README.md

```diff
@@ -56,3 +56,11 @@ yarn lint
 Changed files are auto-linted on commit.
 
 [Set up your editor to automatically lint on save](https://prettier.io/docs/en/editors.html)
+
+## Running Storybook
```

Comment:
Right now only dev has storybook - do we want it to run on `staging` or `test`?

Reply:
I think so? If it's helpful for other teammates to be able to view it (yes?) they shouldn't have to set up a dev environment to do that. If it's easy to run it on both test and staging that's great, otherwise just staging seems fine.